core/src/test/scala/akka/persistence/dynamodb/query/EventsBySliceBacktrackingSpec.scala

```diff
@@ -0,0 +1,208 @@
+    .readJournalFor[DynamoDBReadJournal](DynamoDBReadJournal.Identifier)
+  private val stringSerializer = SerializationExtension(system).serializerFor(classOf[String])
+  private val log = LoggerFactory.getLogger(getClass)
+
+  "eventsBySlices backtracking" should {
+
+    "find old events with earlier timestamp" in {
+      // this scenario is handled by the backtracking query
+      val entityType = nextEntityType()
+      val pid1 = nextPersistenceId(entityType)
+      val slice = query.sliceForPersistenceId(pid1.id)
+      val pid2 = randomPersistenceIdForSlice(entityType, slice)
+      val sinkProbe = TestSink[EventEnvelope[String]]()(system.classicSystem)
+
+      // don't let behind-current-time be a reason for not finding events
+      val startTime = InstantFactory.now().minusSeconds(10 * 60)
+
+      writeEvent(slice, pid1, 1L, startTime, "e1-1")
+      writeEvent(slice, pid1, 2L, startTime.plusMillis(1), "e1-2")
+
+      val result: TestSubscriber.Probe[EventEnvelope[String]] =
+        query
+          .eventsBySlices[String](entityType, slice, slice, NoOffset)
+          .runWith(sinkProbe)
+          .request(100)
+
+      val env1 = result.expectNext()
+      env1.persistenceId shouldBe pid1.id
+      env1.sequenceNr shouldBe 1L
+      env1.eventOption shouldBe Some("e1-1")
+      env1.source shouldBe EnvelopeOrigin.SourceQuery
+
+      val env2 = result.expectNext()
+      env2.persistenceId shouldBe pid1.id
+      env2.sequenceNr shouldBe 2L
+      env2.eventOption shouldBe Some("e1-2")
+      env2.source shouldBe EnvelopeOrigin.SourceQuery
+
+      // first backtracking query kicks in immediately after the first normal query has finished
+      // and it also emits duplicates (by design)
+      val env3 = result.expectNext()
+      env3.persistenceId shouldBe pid1.id
+      env3.sequenceNr shouldBe 1L
+      env3.source shouldBe EnvelopeOrigin.SourceBacktracking
+      // event payload isn't included in backtracking results
+      env3.eventOption shouldBe None
+      // but it can be lazy loaded
+      // FIXME query.loadEnvelope[String](env3.persistenceId, env3.sequenceNr).futureValue.eventOption shouldBe Some("e1-1")
+      // backtracking up to (and equal to) the same offset
+      val env4 = result.expectNext()
+      env4.persistenceId shouldBe pid1.id
+      env4.sequenceNr shouldBe 2L
+      env4.eventOption shouldBe None
+
+      result.expectNoMessage(100.millis) // not e1-2
+
+      writeEvent(slice, pid1, 3L, startTime.plusMillis(3), "e1-3")
+      val env5 = result.expectNext()
+      env5.persistenceId shouldBe pid1.id
+      env5.sequenceNr shouldBe 3L
+
+      // before e1-3 so it will not be found by the normal query
+      writeEvent(slice, pid2, 1L, startTime.plusMillis(2), "e2-1")
+
+      // no backtracking yet
+      result.expectNoMessage(settings.querySettings.refreshInterval + 100.millis)
+
+      // after 1/2 of the backtracking widow, to kick off a backtracking query
```

Comment:
Just tests, but while noticed

```suggestion
      // after 1/2 of the backtracking window, to kick off a backtracking query
```